src/utils/name-utils.ts

```diff
@@ -0,0 +1,13 @@
+import { get } from 'lodash';
+import { BoundaryLayerProps } from '../config/types';
+
+export function getLocationName(
+  layer: BoundaryLayerProps,
+  featureBoundary: any,
```

Comment:
Should avoid `any` use at all costs. Can't you use `Feature<Geometry, GeoJsonProperties>`? If there's a type we can't match it should be defined manually or with `unknown`

Reply:
thanks! done